backend/db/refresh.py

```diff
@@ -124,6 +128,7 @@ def refresh_db(
     try:
         with get_db_connection(local=local) as con:
             since: str = _calc_refresh_datetime(con, since, buffer_hours, force_non_contiguity)
+            since = since if not cset_ids else None
```

Comment:
Bug fix: Was not setting 'since' to None if cset-ids was present.

If both are passed, then a `NotImplementedError` would've gotten raised further down.